### ISSUE TITLE
Migrate from XPBD to Avian if Avian is ready

### ISSUE THREAD

**Issue by JelaMiraj** [TheGrimsey/oxidized_navigation#30]
Greetings, as xpbd is being discontinued, I've been migrating my new foxtrot template (recommended to me by a video helping folks get started on the Bevy Game Jam) based app to avian and Oxidized Navigation (& therefore the template too) is the last dependency I have that doesn't have an avian based version. Thanks for putting in the effort to migrate when you do! :D

**Comment by TheGrimsey** [comment]
Next version will drop support for XPBD and switch to Avian instead.

**Comment by Shatur** [comment]
@TheGrimsey any chance you could take a look into it?

**Comment by TheGrimsey** [comment]
Yes, I admittedly have been focusing on other projects but I just committed switching to Avian. WIll get a release out as well.

**Comment by Shatur** [comment]
Thank you a lot!
Just migrated, can confirm that everything works as expected.

**Comment by TheGrimsey** [comment]
This was solved with the 0.12 release, the crate now uses Avian
